quarkus-workshop-super-heroes/docs/src/docs/asciidoc/optional-cli/cli.adoc

```diff
@@ -8,17 +8,18 @@ You will download the load application, install it and run it.
 
 == Give me some load!
 
-In the `super-heroes/load-super-heroes` directory, there is an application that is **NOT** a Quarkus application.
-It's a simple Java app that simulates users interacting with the system so it generates some load.
+In the `super-heroes/cli-super-heroes` directory, there is a command line Quarkus application .
+This application simulates users interacting with the system so it generates some load.
 
```

Comment:
I don't have very strong feelings about this, but I think I preferred the old directory name, since it made the purpose clearer. The 'cli' name is more about the 'how' than the 'what'. In a sample, focussing on the how is sometimes the right thing, of course.

Reply:
I can come back to the old directory name for sure